effects/letter-clock/data/palette.txt

```diff
@@ -0,0 +1,9 @@
+0b1000 0b10000 0b11000
```

Comment:
I guess it'd be easier if you use `csv` file format - then you'd get parser for free. Please note you can use space as a separator. For clarity I'd add two words in the first line: `orig` and `target`.